### PR TITLE
Add Android example project

### PR DESCRIPTION
**Pull request by NinoDLC**
As an Android developper with limited Gradle skills, I had a lot of issues when trying to migrate from 0.5.0 to 0.6.0. See https://github.com/Kotlin/kotlinx-kover/issues/231 for example. 

I created an example with a very simple Android project. This would make using Kover in Android projects much easier.

### CODE REVIEW

File: examples/android_groovy/app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <!-- Secondary brand color. -->
+        <item name="colorSecondary">#FF03DAC5</item>
+        <item name="colorSecondaryVariant">#FF018786</item>
+        <item name="colorOnSecondary">#FFFFFFFF</item>
+        <!-- Status bar color. -->
+        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>
+    </style>
+
+<!-- TODO NINO    <color name="colorOnBackground">@color/black</color>-->
+
```
Comment (NinoDLC) [inline examples/android_groovy/app/src/main/res/values/themes.xml:17]:
```suggestion

<!-- TODO NINO    <color name="colorOnBackground">@color/black</color>-->

```

**Review by shanshin — COMMENTED** [review]
At the moment, Android examples do not run automatically, so could you in class `ExamplesTests` in the annotation `@ExamplesTest` exclude this example from autorun in this way `@ExamplesTest(excludes = ["android_groovy"])`

File: README.md [hunk 2]
```diff
@@ -188,24 +188,47 @@ android {
 <details>
 <summary>Groovy</summary>
 
+`build.gradle` (Project)
```
[the remark this answers is not in the document]
Comment (NinoDLC) [reply README.md:229]: I learned a bit of gradle.kts so I made another android example in kts this time, and updated both parts (kts and groovy)!

**Comment by NinoDLC** [comment]
I added an example in kts so no Android dev is left behind! :D

File: src/functionalTest/kotlin/kotlinx/kover/test/functional/cases/ExamplesTests.kt [hunk 3]
```diff
@@ -12,7 +12,7 @@ import kotlin.test.*
 
 
 internal class ExamplesTests {
-    @ExamplesTest
+    @ExamplesTest(excludes = ["android_groovy"])
```
Comment (shanshin) [inline src/functionalTest/kotlin/kotlinx/kover/test/functional/cases/ExamplesTests.kt:15]:
```suggestion
    @ExamplesTest(excludes = ["android_groovy", "android_kts"])
```

File: README.md [hunk 4]
```diff
@@ -164,48 +164,109 @@ Example of configuring test task for build type `debug` in Android:
 <details open>
 <summary>Kotlin</summary>
 
+`build.gradle.kts` (Project)
```
Comment (shanshin) [inline README.md:167]: This example is located in the section `Configuration / Configuring JVM test task`, I think it's worth creating a new section `Configuring Android application`, and leave this code as it is.

Comment (NinoDLC) [reply README.md:170]: Done

**Review by shanshin — APPROVED** [review]
Great! Thanks for contribution!

**Pull request merged**